modules/gapi/misc/python/python_bridge.hpp

```diff
@@ -190,6 +192,134 @@ class GAPI_EXPORTS_W_SIMPLE GArrayT
     Storage m_arg;
 };
 
+namespace gapi {
+
+class GAPI_EXPORTS_W_SIMPLE GOutputs
+{
+public:
+    GOutputs() = default;
+    GOutputs(const std::string& id, cv::GKernel::M outMeta, cv::GArgs &&ins);
+
+    GAPI_WRAP cv::GMat     getGMat();
+    GAPI_WRAP cv::GScalar  getGScalar();
+    GAPI_WRAP cv::GArrayT  getGArray(cv::gapi::ArgType type);
+    GAPI_WRAP cv::GOpaqueT getGOpaque(cv::gapi::ArgType type);
+
+private:
+    class Priv;
+    std::shared_ptr<Priv> m_priv;
+};
+
+GOutputs op(const std::string& id, cv::GKernel::M outMeta, cv::GArgs&& args);
+
+template <typename... T>
+GOutputs op(const std::string& id, cv::GKernel::M outMeta, T&&... args)
+{
+    return op(id, outMeta, cv::GArgs{cv::GArg(std::forward<T>(args))... });
+}
+
+} // namespace gapi
 } // namespace cv
 
+cv::gapi::GOutputs cv::gapi::op(const std::string& id,
+                                cv::GKernel::M outMeta,
+                                cv::GArgs&& args)
+{
+    cv::gapi::GOutputs outputs{id, outMeta, std::move(args)};
+    return outputs;
+}
+
+class cv::gapi::GOutputs::Priv
```

Comment:
Priv is usually hidden from the headers (as it acts for PIMPL purposes)

Reply:
There is no way to implement it in `cpp` file, because this files will be included into `cv2.cpp`